### ISSUE TITLE
Error with GtkReactive and GUI mousse info 

### ISSUE THREAD

**Issue by RGerzaguet** [timholy/ProfileView.jl#112]
Hello,

First shoutout for ProfileView() (and let's be honest shoutout for everything you do related to Julia !).
I have an issue with ProfileView recently (I would say since I switch to 1.1 but I am not really sure about the schedule) when I try to get information from the TgK display. You can see below the message error. It happens every time I use ProfileView.view() whatever the profile done. The Julia process itself freeze and crash when I close the GUI window.

I am quite sure that the issue comes from the GUI itself and not ProfileView but I struggle to find any fix. Any suggestion  ? I have found something related in https://github.com/JuliaLang/julia/issues/21141 but this is quite unclear for me. 

Some infos: 
- MacBook Pro (13-inch, 2017, Four), MacOS Moyave 
- Julia 1.1 
```Julia 
julia> versioninfo()
Julia Version 1.1.0
Commit 80516ca202 (2019-01-21 21:24 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i7-7567U CPU @ 3.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
```
- Packages version 
```
 [c46f51b8] ProfileView v0.4.0
 [159f3aea] Cairo v0.5.6
```

The log of the error (happen as soon as I click anywhere on the GUI) 
```
julia> Failed to push!
    GtkReactive.MouseButton{GtkReactive.UserUnit}(XY{UserUnit}(8352.453076120959, 2.1045258620689657), 0x00000000, 3, 0, nothing)
to node
    3: "input-3" = GtkReactive.MouseButton{GtkReactive.UserUnit}(XY{UserUnit}(8352.453076120959, 2.1045258620689657), 0x00000000, 3, 0, nothing) GtkReactive.MouseButton{GtkReactive.UserUnit} (active)

error at node: 29: "map(input-3)" = nothing Nothing (active)
DivideError: integer division error
show_layout at /Users/home/.julia/packages/Cairo/CXPG1/src/Cairo.jl:1131 [inlined]
#text#2(::String, ::String, ::Int64, ::Bool, ::Function, ::Cairo.CairoContext, ::GtkReactive.UserUnit, ::GtkReactive.UserUnit, ::String) at /Users/home/.julia/packages/Cairo/CXPG1/src/Cairo.jl:1206
#text at ./none:0 [inlined]
(::getfield(ProfileView.ProfileViewGtk, Symbol("##10#13")){Int64,GtkReactive.Canvas{GtkReactive.UserUnit},Reactive.Signal{GtkReactive.ZoomRegion{Float64}},Cairo.CairoSurfaceImage{UInt32}})(::GtkReactive.MouseButton{GtkReactive.UserUnit}) at /Users/home/.julia/packages/ProfileView/0MFdX/src/ProfileViewGtk.jl:113
#33 at /Users/home/.julia/packages/Reactive/g1LZ6/src/operators.jl:41 [inlined]
runaction(::getfield(Reactive, Symbol("##33#34")){getfield(ProfileView.ProfileViewGtk, Symbol("##10#13")){Int64,GtkReactive.Canvas{GtkReactive.UserUnit},Reactive.Signal{GtkReactive.ZoomRegion{Float64}},Cairo.CairoSurfaceImage{UInt32}},Reactive.Signal{Nothing},Tuple{Reactive.Signal{GtkReactive.MouseButton{GtkReactive.UserUnit}}}}) at /Users/home/.julia/packages/Reactive/g1LZ6/src/core.jl:313
foreach(::typeof(Reactive.runaction), ::Array{Function,1}) at ./abstractarray.jl:1866
run_node(::Reactive.Signal{Nothing}) at /Users/home/.julia/packages/Reactive/g1LZ6/src/core.jl:340
run_push(::Reactive.Signal{GtkReactive.MouseButton{GtkReactive.UserUnit}}, ::GtkReactive.MouseButton{GtkReactive.UserUnit}, ::typeof(Reactive.print_error), ::Bool) at /Users/home/.julia/packages/Reactive/g1LZ6/src/core.jl:358
run_push(::Reactive.Signal{GtkReactive.MouseButton{GtkReactive.UserUnit}}, ::GtkReactive.MouseButton{GtkReactive.UserUnit}, ::Function) at /Users/home/.julia/packages/Reactive/g1LZ6/src/core.jl:345
run(::Int64) at /Users/home/.julia/packages/Reactive/g1LZ6/src/core.jl:306
run at /Users/home/.julia/packages/Reactive/g1LZ6/src/core.jl:300 [inlined]
(::getfield(Reactive, Symbol("##27#29")))() at ./task.jl:259
```

Thanks a lot !

**Comment by RGerzaguet** [comment]
Me again, 

Sorry for the noise, this issue is related to https://github.com/JuliaGraphics/Cairo.jl/issues/271 and moving to glib 2.58.3 solved the GUI issue. 

Thanks !

**Comment by timholy** [comment]
Glad you figured it out, and thanks!

**Comment by timholy** [comment]
Should be fixed if you use Julia 1.3 and up-to-date packages
